### PR TITLE
Modify doc to include ext folder information

### PR DESCRIPTION
**Pull request by shaneonabike**
It's important to not forget the information about the extension folders

### CODE REVIEW

File: docs/upgrade/wordpress.md [hunk 1]
```diff
@@ -40,6 +40,9 @@ The ["Before upgrading"](/upgrade/index.md#before-upgrading) steps describe step
 
 #### CiviCRM files and database
 1. Backup your existing <wordpress>/wp-content/plugins/civicrm directory by creating an archive or FTPing it to your local computer.
+
+!!! note: It's important at the very least to backup your ext directory or you will need to redownload all the extension
```
Comment (MegaphoneJon) [inline docs/upgrade/wordpress.md]: Could we call it the "extensions" directory?  Not everyone's directory is called "ext", that's a frequently-changed default.  Also I believe it should be "all the extensions.", not "all the extension"?

File: docs/upgrade/wordpress.md [hunk 2]
```diff
@@ -106,6 +109,9 @@ Delete all files in your `templates_c` directory
 !!! caution "For CiviCRM 4.6 and older:"
     `<wordpress_root>/<content-dir>/plugins/files/civicrm/templates_c`
 
+## Copy back your extensions
+1. Restore all the extensions that were backed up from the ext folder into your newly downloaded folder.
```
Comment (MegaphoneJon) [inline docs/upgrade/wordpress.md]: Same /ext/extensions/ change here and I'll merge this.  Thanks!

File: docs/upgrade/wordpress.md (hunk 1 above)
Comment (shaneonabike) [reply docs/upgrade/wordpress.md]: Fair enough I think I was trying to guide them, but you are right perhaps I should use extensions.

Comment (shaneonabike) [reply docs/upgrade/wordpress.md]: Ok I made made the changes

File: docs/upgrade/wordpress.md (hunk 2 above)
Comment (shaneonabike) [reply docs/upgrade/wordpress.md]: I also made the changes here and specified where they could restore the backups thanks for calling me out on being cryptic ;)

**Comment by MegaphoneJon** [comment]
Well - I was confusing this PR and the last one, and thought this was a Drupal-to-WordPress change.  Now that I see what it is...well, I can't say I do this sort of backup, so my opinions aren't very strong :)

That said - I think that rather than telling people to back up a specific directory (which is often changed to work better with git etc.), We should tell them "The path of the extension directory can be found at **Administer menu » System Settings » Directories**".

Other than that, looks good.

**Pull request merged**